LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Ipmi.cs

```diff
@@ -0,0 +1,410 @@
+// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// All Rights Reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Management;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace LibreHardwareMonitor.Hardware.Motherboard.Lpc;
+
+// Ported from ipmiutil
+[StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi, Pack = 1)]
+unsafe struct IpmiSdr
```

Comment:
Can you move native code to Interop folder where it belogns?

Reply:
Wasn't sure about this as it isn't quite the same kind of native as the others in Interop, have moved :)